src/plugins/index_pattern_management/public/components/edit_index_pattern/edit_index_pattern.tsx

```diff
@@ -145,12 +154,17 @@ export const EditIndexPattern = withRouter(
           defaultIndex={defaultIndex}
         >
           {showTagsSection && (
-            <EuiFlexGroup wrap>
+            <EuiFlexGroup wrap gutterSize="s">
               {Boolean(indexPattern.timeFieldName) && (
                 <EuiFlexItem grow={false}>
                   <EuiBadge color="warning">{timeFilterHeader}</EuiBadge>
                 </EuiFlexItem>
               )}
+              {indexPattern.id && indexPattern.id === securitySolution && (
```

Comment:
I had to change the `security-solution` id to be space aware with the saved object unique ids across spaces change. So it will be like  `security-solution-default`: https://github.com/elastic/kibana/pull/114806/files#diff-b60e90929c1abe12e979452580d6456009f23e5abe21648abbe569f292ae19eeR22. 

If you change to this, it should work:

```suggestion
              {indexPattern.id && indexPattern.id.indexOf(securitySolution) === 0 && (
```